pkgs/applications/audio/csound/csound-qt/examplepath.patch

```diff
@@ -0,0 +1,41 @@
+diff --git a/src/qutecsound.cpp b/src/qutecsound.cpp
+index 103c006f..59340962 100644
+--- a/src/qutecsound.cpp
++++ b/src/qutecsound.cpp
+@@ -2385,7 +2385,7 @@ void CsoundQt::handleTableSyntax(QString syntax)
+ }
+ 
+ void CsoundQt::openManualExample(QString fileName)
+-{    
++{
```

Comment:
This change is unnecessary. Same goes for the one just below.

Reply:
I'll remove

Reply:
Done